cmd/manager/main.go

```diff
@@ -69,7 +69,7 @@ func main() {
 		Port:                   9443,
 		HealthProbeBindAddress: probeAddr,
 		LeaderElection:         enableLeaderElection,
-		LeaderElectionID:       "df55ab35.linkall.com",
+		LeaderElectionID:       "1fb9b8ef.vanus.ai",
```

Comment:
why hardcode here?

Reply:
This file is automatically generated by operator-sdk